.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,67 @@
+name: Build and publish scroll-alpine image
+# This workflow aims to build images from the feat-deterministic-deployment branch
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Clone, Build, Publish
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Update submodules recursively
+        run: git submodule update --init --recursive
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v3
+        with:
+          node-version: '21'
+
+      - name: Install dependencies
+        run: npm install
```

Comment:
Not sure we need this, as I remember we install deps inside the container image. But if you tested and it works we can keep it.